wd_util.c

```diff
@@ -312,12 +312,76 @@ void dump_env_info(struct wd_env_config *config)
 	}
 }
 
+static void *wd_get_config_numa(struct wd_env_config *config, int node)
+{
+	struct wd_env_config_per_numa *config_numa;
+	int i;
+
+	FOREACH_NUMA(i, config, config_numa)
+		if (config_numa->node == node)
+			break;
+
+	if (i == config->numa_num)
+		return NULL;
+
+	return config_numa;
+}
+
+static void wd_free_numa(struct wd_env_config *config)
+{
+	struct wd_env_config_per_numa *config_numa;
+	int i;
+
+	if (!config->config_per_numa)
+		return;
+
+	FOREACH_NUMA(i, config, config_numa)
+		free(config_numa->dev);
+
+	free(config->config_per_numa);
+}
+
+static void wd_get_dev_numa(struct uacce_dev_list *head,
+			    int *numa_dev_num, int *numa_num)
```

Comment:
Could you help to share more comments on **numa_dev_num** and **numa_num**?

Reply:
numa_dev_num represents how many device of one type(like sec2 support cipher) on this numa, numa_num represents how many numa has such devices. 
For example, if we are on kunpeng 920, only node 0 and node 2 has sec2 devices,but node 0 may connect with two  sec2 devices,so numa_dev_num is 2, not 1.

Reply:
Good. Please comment it in the code.

Reply:
ok,I will change it later